explainaboard/utils/eval_ops.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import annotations
 
-from collections.abc import Iterator
-
 from datalabs.operations.aggregate.auto_eval import auto_eval
 
 from explainaboard import FileType, get_loader, get_processor, Source
 
 
 @auto_eval(name="explainaboard")
-def explainaboard(samples: Iterator, dataset_info=None):
+def explainaboard(samples: str, dataset_info=None):
+    # TODO(odashi): I didn't understand what this docstring mean.
+    # This function seems to be not used anywhere, so I guessed the file can be removed.
```

Comment:
@neubig please give me details about this.

Reply:
This function could be removed.

Reply:
Removed: 17309a5